packages/backend/src/logger/customLogger.ts

```diff
@@ -42,15 +50,38 @@ const transports = {
       ),
     }),
   ],
-  auditLog: [
-    new winston.transports.Console({
-      format: winston.format.combine(
-        auditLogFormat({ isAuditLog: true }),
-        defaultFormat,
-        winston.format.json(),
-      ),
-    }),
-  ],
+  auditLog: (config?: Config) => {
+    if (config?.getOptionalBoolean('console.enabled') === false) {
```

Comment:
```suggestion
    if (!config?.getOptionalBoolean('console.enabled')) {
```
For uniformity with the other function.

Reply:
I did that on purpose to ensure that it will be on by default unless explicitly turned off by `console.enabled=false`. If we changed it to that, then not providing any configs would result in no audit logs. Do we want to disable audit logs by default?

Reply:
We should keep at least one transport enabled by default so we shouldn't make that chage